ai_diffusion/workflow.py

```diff
@@ -634,10 +653,16 @@ def apply_edit_conditioning(
 
     extra_input = [c.image for c in control_layers if c.mode.is_ip_adapter]
     if len(extra_input) == 0:
+        if arch == Arch.qwen_e:
+            # Don't use VAE to force the reference latent
+            cond = w.text_encode_qwen_image_edit(clip, None, input_image, positive)
```

Comment:
If you don't pass VAE, does passing the input image even do anything? Is it actually any different than regular text-encode node?